pyro/contrib/examples/finance.py

```diff
@@ -0,0 +1,25 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import urllib
+
+import pandas as pd
+
+from pyro.contrib.examples.util import get_data_directory
+
+DATA = get_data_directory(__file__)
+
+# https://finance.yahoo.com/quote/%5EGSPC/history/
+CACHE_URL = "https://fritzo.org/media/snp500.csv.bz2"
```

Comment:
@neerajprad Could you upload this to our CDN and either update this url or comment so I can update this url?

Reply:
The dataset is at - https://d2hg8soec8ck9v.cloudfront.net/datasets/snp500.csv.bz2

Reply:
Thanks!